.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
```

Comment:
Can you remove this file please? I suspect it was added before the .gitignore change.